### PR TITLE
Added quote computations

### PR DESCRIPTION
**Pull request by ajlane17**

### CODE REVIEW

File: CIT365_W9_MegaDeskV2/Pages/DeskQuotes/Create.cshtml.cs [hunk 1]
```diff
@@ -8,39 +8,43 @@
...
 
 namespace CIT365_W9_MegaDeskV2.Pages.DeskQuotes
 {
-    public class CreateModel : PageModel
+    public class CreateModel : DeskQuotePageModel
     {
         private readonly CIT365_W9_MegaDeskV2.Data.CIT365_W9_MegaDeskV2Context _context;
+        private readonly IConfiguration _configuration;
 
         public List<SelectListItem> Countries { get; } = new List<SelectListItem>
```
Comment (crosby-dan) [inline CIT365_W9_MegaDeskV2/Pages/DeskQuotes/Create.cshtml.cs:21]: This code was accidentally left in by me.

Comment (ajlane17) [reply CIT365_W9_MegaDeskV2/Pages/DeskQuotes/Create.cshtml.cs:21]: Hey Dan! I'm actually using the configuration to pull in the pricing data, see https://github.com/ajlane17/cit365-group/pull/3/files#diff-d35cf59d0706e7805bc0a3daf5616e27 for details - dumb the link to the appsettings diff, doesn't work, but anyway, that's the file with the relevant data.

File: CIT365_W9_MegaDeskV2/Pages/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -2,8 +2,8 @@
 <html lang="en">
 <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>@ViewData["Title"] - CIT365_W9_MegaDeskV2</title>
+    <meta name="viewport" content="wIdth=device-wIdth, initial-scale=1.0" />
```
Comment (crosby-dan) [inline CIT365_W9_MegaDeskV2/Pages/Shared/_Layout.cshtml]: upper case "I" in width

Comment (ajlane17) [reply CIT365_W9_MegaDeskV2/Pages/Shared/_Layout.cshtml]: huh! Great catch! I'm not even sure how that got in there!! Must have been typo in a naming refactor and I missed that when reviewing changes. Will fix

**Pull request merged**